### ISSUE TITLE
not able to access 'gs://practical-ml-vision-book/flowers_tfr/train'

### ISSUE THREAD

**Issue by alex55711** [GoogleCloudPlatform/practical-ml-vision-book#13]
Hello, 
this was working for me until this morning.
Has this gs bucket access being disabled?

Please advise.

regards,

Alejandro H.

**Comment by AdrienLemaire** [comment]
Same here, unable to complete the "Classifying Images with a Linear Model" [qwiklabs](https://googlecoursera.qwiklabs.com/focuses/29098178?parent=lti_session)  because the bucket access is forbidden.

```console
!gsutil cat gs://practical-ml-vision-book/flowers_5_jpeg/flower_photos/train_set.csv | head -20

AccessDeniedException: 403 There is an account problem for the requested project.
```

**Comment by casualcomputer** [comment]
I got exactly the same error as @AdrienLemaire

**Comment by alex55711** [comment]
This issue went away and now im able to access the bucket.
I think there is some maintanance going on the books GS buckets that prevented me to access.
@AdrienLemaire  @casualcomputer  you should check too if now you dont have an issue.
![image](https://github.com/GoogleCloudPlatform/practical-ml-vision-book/assets/132787510/6d2a8697-4845-4237-93c6-22069a3e4b74)

**Comment by hookmax** [comment]
The issue still exists:
!gsutil cat gs://practical-ml-vision-book/flowers_5_jpeg/flower_photos/train_set.csv | head -20
AccessDeniedException: 403 236450629614-compute@developer.gserviceaccount.com does not have storage.objects.list access to the Google Cloud Storage bucket. Permission 'storage.objects.list' denied on resource (or it may not exist).

**Comment by Digital1O1** [comment]
> ```
> !gsutil cat gs://practical-ml-vision-book/flowers_5_jpeg/flower_photos/train_set.csv | head -20
> ```

I'm in the same boat as of right now

**Comment by sergani** [comment]
Hi guys, I'm getting the same error and unable to access the bucket to access the images and complete the lab. Was anyone able to resolve this?

**Comment by Digital1O1** [comment]
> Hi guys, I'm getting the same error and unable to access the bucket to access the images and complete the lab. Was anyone able to resolve this?

I personally gave up hope on the matter and started to just find datasets on places like Kaggle so I can keep moving forward with the book since I'm super skeptical the authors check the github repository

**Comment by takumiohym** [comment]
Hi all, the bucket became available again. Please rerun the code. Thanks!

**Comment by sergani** [comment]
Hi @takumiohym , thank you for the support. I confirm access to the bucket is working fine now.

**Comment by jimxu0621** [comment]
testing 06a_resizing.ipynb and get a similar error code 403 and error message

---------------------------------------------------------------------------
PermissionDeniedError                     Traceback (most recent call last)
[<ipython-input-7-2550db9863bd>](https://localhost:8080/#) in <cell line: 16>()
     15 
     16 train_dataset = tf.data.TFRecordDataset(
---> 17     [filename for filename in tf.io.gfile.glob(
     18         'gs://practical-ml-vision-book/flowers_tfr/train-*')
     19     ], compression_type='GZIP').map(parse_tfr)

PermissionDeniedError: Error executing an HTTP request: HTTP response code 403 with body '{
  "error": {
    "code": 403,
    "message": "[xxx@gmail.com](mailto:xxx@gmail.com) does not have storage.objects.list access to the Google Cloud Storage bucket. Permission 'storage.objects.list' denied on resource (or it may not exist).",

**Comment by takumiohym** [comment]
Thanks for reporting @jimxu0621
I'll try to restore the access, but meanwhile could you try again changing the bucket path to `gs://practical-ml-vision-book-data` adding `-data`?

**Comment by jimxu0621** [comment]
> Thanks for reporting [@jimxu0621](https://github.com/jimxu0621) I'll try to restore the access, but meanwhile could you try again changing the bucket path to `gs://practical-ml-vision-book-data` adding `-data`?

Thank you, that part works after adding -data, I guess I should make the same changes for all other notebooks in chapter 6 when I test.

**Comment by takumiohym** [comment]
Yes. Please let me know if you see any other issues!

**Comment by AnkitaBeniwal** [comment]
I tried to add '-data' but still the error persists as the service account is still using without '-data'. 
Could you please notify when the bucket is active again!?

**Comment by takumiohym** [comment]
Which notebook are you referring to? @AnkitaBeniwal

**Comment by AnkitaBeniwal** [comment]
> Which notebook are you referring to? [@AnkitaBeniwal](https://github.com/AnkitaBeniwal)

classifying_images_with_transfer_learning.ipynb

**Comment by takumiohym** [comment]
Is it under this repository? Could you share the link? @AnkitaBeniwal

**Comment by AnkitaBeniwal** [comment]
> Is it under this repository? Could you share the link? [@AnkitaBeniwal](https://github.com/AnkitaBeniwal)

https://github.com/GoogleCloudPlatform/training-data-analyst/blob/master/courses/machine_learning/deepdive2/computer_vision_fun/labs/classifying_images_with_transfer_learning.ipynb
Thank you for replying promptly!

**Comment by takumiohym** [comment]
It seems it's not under this repository, but I addressed the issue. Could you rerun it again?

**Comment by AnkitaBeniwal** [comment]
> It seems it's not under this repository, but I addressed the issue. Could you rerun it again?

Thanks a million. Apologies for reporting the error here. I have been facing this error for almost all the notebooks in this repository quoted above.

**Comment by takumiohym** [comment]
Opened PRs on both repositories to update to the new path @AnkitaBeniwal @jimxu0621 
Thanks for reporting the issue!

**Comment by mjmortega** [comment]
Can anyone access the `train_set.csv` and `eval_set.csv` from **02a_machine_perception.ipynb** ?

**Comment by takumiohym** [comment]
@mjmortega Could you make sure you are using the latest notebook? The path is updated in January.

**Comment by mjmortega** [comment]
> [@mjmortega](https://github.com/mjmortega) Could you make sure you are using the latest notebook? The path is updated in January.

Yes, I'm pretty sure I used the latest notebook in this repository.

**Comment by takumiohym** [comment]
Then yes, I can access to the files.

**Comment by techonn** [comment]
Hi it seems like this is still an issue. I also get "Forbidden Error 403" when I click on the link to `train_set.csv`.

<img width="1078" height="490" alt="Image" src="https://github.com/user-attachments/assets/b214bbf5-4e63-4c3f-8157-d5c1a70c52f6" />
